lib/scala/searcher/src/main/resources/reference.conf

```diff
@@ -4,6 +4,7 @@ searcher {
     driver = "org.sqlite.JDBC"
     connectionPool = disabled
     properties.journal_mode = "wal"
+    properties.locking_mode = "EXCLUSIVE"
```

Comment:
```suggestion
```

Tests are using `BaseServerTest#getInitializedWsClient()` method that creates a brand new client controller (including a database instance) every time. With the exclusive locking, the first call locks the database, and consequent calls will be waiting for the lock, making the tests to fail with a timeout.

You should either remove the setting from the default config or disable exclusive locking in tests.